server/graphql/index.ts

```diff
@@ -85,6 +85,46 @@ export const rootValue = {
           space
         };
       });
+    },
+    votes: async (
+      parent,
+      { 
+        id,
+        first = 10,
+        skip = 0,
+        address,
+        space
```

Comment:
This should be "spaces" array (same as timeline query). Also can you add an argument "proposal"? This way we can use this endpoint on the proposal page to replace the Rest API.